package-lock.json

```diff
@@ -1510,6 +1510,12 @@
         "unist-util-find-all-after": "^3.0.2"
       }
     },
+    "@tailwindcss/aspect-ratio": {
```

Comment:
Hi @fessehaye! It turns out Tailwind has a aspect-ration package that we can use instead of bootstraps. Just wanted to double check if I was adding it to package-lock.json correctly 😄 thanks!